pages/api/bitbucket/callbacks/webhook.ts

```diff
@@ -1,30 +1,43 @@
 import { NextApiRequest, NextApiResponse } from 'next';
 import { publishMessage } from '../../../../utils/pubsub/pubsubClient';
 import { getTopicNameFromDB } from '../../../../utils/db/relevance';
+import { getRepoConfig } from '../../../../utils/db/repos';
 
 const webhookHandler = async (req: NextApiRequest, res: NextApiResponse) => {
   const jsonBody = req.body;
   const owner = jsonBody.repository.owner.username; 
   const provider = "bitbucket";
   const name = jsonBody.repository.name;
-  const topicName = await getTopicNameFromDB(owner, name, provider);
-  
+  const topicName = await getTopicNameFromDB(owner, name, provider).catch((error) => {
+    console.error('Failed to get repoConfig from db :', error);
+    res.status(500).json({ error: 'Internal Server Error' });
+  });
+  const repoConfig = await getRepoConfig({
+    repo_provider: provider,
+    repo_owner: owner,
+    repo_name: name
+  }).catch((error) => {
+    console.error('Failed to get repoConfig from db :', error);
+    res.status(500).json({ error: 'Internal Server Error' });
```

Comment:
```suggestion
    return res.status(500).json({ error: 'Internal Server Error' });
```

The rabbit is correct again, although, our convention is to use the return statement with the `res.json` call itself.
(Although, now that I think about it, it might not be optimal for reading).